examples/ts/eth/decodeSignature.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Recover the signing address of a signature
```

Comment:
May need to specify that its a function to recover the signer address of the siganture payload for a eth multisig transaction